src/codeu/chat/server/Server.java

```diff
@@ -66,46 +62,47 @@
   private final Relay relay;
   private Uuid lastSeen = Uuid.NULL;
 
-  private static ServerInfo info;
-
   public Server(final Uuid id, final Secret secret, final Relay relay) {
 
     this.id = id;
     this.secret = secret;
     this.controller = new Controller(id, model);
     this.relay = relay;
 
-    info = new ServerInfo();
-
     this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command() {
-      public void onMessage(InputStream in, OutputStream out) throws IOException {
+      public void onMessage(InputStream in, OutputStream out)
+          throws IOException {
         Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
-        Uuid.SERIALIZER.write(out, info.version);
+        ServerInfo.SERIALIZER.write(out, info);
       }
     });
 
     // New Message - A client wants to add a new message to the back end.
     this.commands.put(NetworkCode.NEW_MESSAGE_REQUEST, new Command() {
       @Override
-      public void onMessage(InputStream in, OutputStream out) throws IOException {
+      public void onMessage(InputStream in, OutputStream out)
+          throws IOException {
 
         final Uuid author = Uuid.SERIALIZER.read(in);
         final Uuid conversation = Uuid.SERIALIZER.read(in);
         final String content = Serializers.STRING.read(in);
 
-        final Message message = controller.newMessage(author, conversation, content);
+        final Message message = controller.newMessage(author, conversation,
+            content);
```

Comment:
Oh. Interesting. I always had it setup at 80 for C++ (following the google style guide). I didn't realize that they were different for java.